operator/api/redpanda/v1alpha2/zz_generated_status.go

```diff
@@ -0,0 +1,258 @@
+	// This reason is used with the "Bound" condition when it evaluates to True
+	// because a node pool is bound to a cluster.
+	NodePoolBoundReasonBound NodePoolBoundCondition = "Bound"
+	// This reason is used with the "Bound" condition when it evaluates to False
+	// because a node pool is not bound to a cluster.
+	NodePoolBoundReasonNotBound NodePoolBoundCondition = "NotBound"
+	// This reason is used when a node pool has only been partially reconciled and
+	// we have early returned due to a retryable error occurring prior to applying
+	// the desired node pool state.
+	NodePoolBoundReasonError NodePoolBoundCondition = "Error"
+	// This reason is used when a node pool has only been partially reconciled and
+	// we have early returned due to a known terminal error occurring prior to
+	// applying the desired node pool state.
+	NodePoolBoundReasonTerminalError NodePoolBoundCondition = "TerminalError"
+
+	// This reason is used with the "Deployed" condition when it evaluates to True
+	// because a node pool has been fully deployed for a cluster.
+	NodePoolDeployedReasonDeployed NodePoolDeployedCondition = "Deployed"
+	// This reason is used with the "Deployed" condition when it evaluates to False
+	// because a node pool has not yet been fully deployed for a cluster.
+	NodePoolDeployedReasonScaling NodePoolDeployedCondition = "Scaling"
+	// This reason is used with the "Deployed" condition when it evaluates to False
+	// because a node pool has not started to deploy for a cluster.
+	NodePoolDeployedReasonNotDeployed NodePoolDeployedCondition = "NotDeployed"
+	// This reason is used when a node pool has only been partially reconciled and
+	// we have early returned due to a retryable error occurring prior to applying
+	// the desired node pool state.
+	NodePoolDeployedReasonError NodePoolDeployedCondition = "Error"
+	// This reason is used when a node pool has only been partially reconciled and
+	// we have early returned due to a known terminal error occurring prior to
+	// applying the desired node pool state.
+	NodePoolDeployedReasonTerminalError NodePoolDeployedCondition = "TerminalError"
+
+	// This reason is used with the "Quiesced" condition when it evaluates to True
+	// because the operator has finished reconciling the node pool at its current
+	// generation.
+	NodePoolQuiescedReasonQuiesced NodePoolQuiescedCondition = "Quiesced"
+	// This reason is used with the "Quiesced" condition when it evaluates to False
+	// because the operator has not finished reconciling the node pool at its
+	// current generation.
+	NodePoolQuiescedReasonStillReconciling NodePoolQuiescedCondition = "StillReconciling"
+
+	// This reason is used with the "Stable" condition when it evaluates to True
+	// because all dependent conditions also evaluate to True.
+	NodePoolStableReasonStable NodePoolStableCondition = "Stable"
+	// This reason is used with the "Stable" condition when it evaluates to True
```

Comment:
Seems like a typo. The “Stable” condition evaluates to True even when a dependent condition is False?